.circleci/config.yml

```diff
@@ -0,0 +1,157 @@
+version: 2.1
+
+orbs:
+  cypress: cypress-io/cypress@1.27.0
+  mac: circleci/macos@1.1.0
+  node: circleci/node@4.2.1
+  win: circleci/windows@2.4.0
+
+executors:
+  linux:
+    docker:
+      - image: circleci/buildpack-deps:trusty
+        user: root
+  macos:
+    macos:
+      xcode: 11.4
+  win: 
+    machine:
+      image: windows-server-2019-vs2019:stable
+      resource_class: windows.medium
+      shell: powershell.exe
+
+anchors:
+  - &npm_cache_key package-cache-{{ .Branch }}-{{ checksum "package-lock.json" }}
```

Comment:
I recently added a `{{ .Environment.CACHE_VERSION }}` to Getty.edu so that we have a mechanism to manually invalid the package cache in CircleCI.